sphinx/source/docs/user_guide/interaction.rst

```diff
@@ -353,6 +353,64 @@ hover tool is over.
 
     show(p)
 
+  Callbacks for Range Update
```

Comment:
I would add a

`.. _userguide_interaction_actions_range_callbacks:`

before the title to have it available to link it later...